### ISSUE TITLE
Upgrade OpenShift version

### ISSUE THREAD

**Issue by bvkin** [redhat-cop/katacoda-scenarios#18]
Currently, our katacoda scenarios are running OpenShift 3.6. Ideally we should upgrade this, probably to 3.11.

From katacoda:
```
$ oc version
oc v3.6.0+c4dd4cf
kubernetes v1.6.1+5115d708d7
features: Basic-Auth GSSAPI Kerberos SPNEGO
```

**Comment by pcarney8** [comment]
@bvkin agreed! unfortunately the open-source katacoda only support that image. We need to figure out who runs the learn.openshift.com one and get this onto there! Newer versions, etc

**Comment by InfoSec812** [comment]
@pcarney8 @bvkin Let me know if adding my voice will help make it loud enough to get some movement on this.
